core/src/test/scala/org/allenai/common/EnumSpec.scala

```diff
@@ -4,15 +4,11 @@ import org.allenai.common.testkit.UnitSpec
 
 import spray.json._
 
-class EnumSpec extends UnitSpec {
+import java.io.{ FileOutputStream, FileInputStream }
```

Comment:
Drive by style comment: I rely on Intellij to manage imports. The only
thing I do manually is to remove unused imports, conveniently marked in
grey. In XX years of Java / Scala, I never cared once about the order of
imports; I blank them out harder than ads on the Web. I get it's nice to
have tidy imports, but barring a good autoformat, should we spend
nano-attention bits on this?

On Wed, Nov 18, 2015 at 8:06 AM, Mark Schaake <notifications@github.com>
wrote:

> In core/src/test/scala/org/allenai/common/EnumSpec.scala
> <https://github.com/allenai/common/pull/159#discussion_r45218952>:
>
> > @@ -4,15 +4,11 @@ import org.allenai.common.testkit.UnitSpec
> >
> >  import spray.json._
> >
> > -class EnumSpec extends UnitSpec {
> > +import java.io.{ FileOutputStream, FileInputStream }
>
> they are in the import block. Are you misreading the diff?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/allenai/common/pull/159/files#r45218952>.
>